### ISSUE TITLE
Ambilight Effect Type: Vertical Average Color

### ISSUE THREAD

**Issue by dondonini** [antonpup/Aurora#1936]
Instead of mirroring the entire screen/region onto the keyboard or averaging the entire area, only average the vertical pixels of the screen/region onto the keyboard.
Similar to Ambilight strips on TVs like this:
![image](https://user-images.githubusercontent.com/18273458/77240728-7d19f480-6bbf-11ea-8c3a-8c8894b181b4.png)

For example, this is how my screen looks like right now:
![image](https://user-images.githubusercontent.com/18273458/77240329-0a0e7f00-6bbb-11ea-95ef-90eeb5c41c84.png)

and I managed to make it work with 5 layers, coordinates and key/region exclusion (Took me half an hour to just do 5 zones)
![image](https://user-images.githubusercontent.com/18273458/77240355-6ec9d980-6bbb-11ea-9d04-94bf6cb56862.png)

Here's a crude paint.NET diagram of what I'm trying to do:
![image](https://user-images.githubusercontent.com/18273458/77240576-05979580-6bbe-11ea-941e-94f2938dddb2.png)

So basically, what I'm doing is creating regions on the screen, getting the average colour in that region and applying it on the zone on the keyboard/mouse.

My suggestion is to have a full-res version of what I just did instead of just 5 zones.
